includes/cenchat-meta.php

```diff
@@ -0,0 +1,16 @@
+<?php
+class Cenchat_Meta {
+
+    public function add_meta() {
+        $options = get_option( 'cenchat_id' );
+        ?>
+        <meta property="cenchat:id" content="<?php echo $options['id_text']; ?>">
```

Comment:
Can we move this to `public/partials/` folder? I think a regular function that accepts the `$options['idx_text']` would suffice.